userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -159,11 +187,15 @@ void cri::set_cri_timeout(int64_t timeout_ms)
 	s_cri_timeout = timeout_ms;
 }
 
+void cri::set_extra_queries(bool extra_queries) {
+	s_cri_extra_queries = extra_queries;
+}
+
 bool cri::resolve(sinsp_container_manager* manager, sinsp_threadinfo* tinfo, bool query_os_for_missing_info)
 {
 	sinsp_container_info container_info;
 
-	if(docker::detect_docker(tinfo, container_info.m_id))
+	if(matches_runc_cgroups(tinfo, CRI_CGROUP_LAYOUT, container_info.m_id))
 	{
 		container_info.m_type = s_cri_runtime_type;
```

Comment:
Is the need here only to specify if it is a CRI-based run-time or not ? Do we not need to know if it is `CT_CRIO` or 'CT_CONTAINERD` ?